tables/tr-g1.tbl

```diff
@@ -0,0 +1,18 @@
+#-display-name: Türkçe 1. derece Çıktı Kısaltmalı Braille Tablosu
```

Comment:
`#-display-name` should be English (and don't forget to update extra/generate-display-names/display-names). `#-name` can be used for the Turkish name. Apparently same problem in tr.tbl.